addons/api/mirage/serializers/role.js

```diff
@@ -38,11 +38,6 @@ export default ApplicationSerializer.extend({
 
     if (!json.principals.length) delete json.principals;
 
-    // default grant scopes
```

Comment:
Now that we can simply define the `grant_scope_ids` as `['this']` we don't need this serializer code anymore.